.env.sample

```diff
@@ -1,3 +1,4 @@
 PROVIDER_URL=http://localhost:8545
 IPFS_GATEWAY_URL=https://ipfs.io
+COVER_ROUTER_URL=https://localhost:5001
 PORT=3000
```

Comment:
Might be out of scope, but can we name this `.env.example` instead? It's more of a standard which some tooling picks up